lib/cocoapods/installer/xcode/pods_project_generator/pod_target_integrator.rb

```diff
@@ -58,11 +58,7 @@ def add_copy_resources_script_phase(native_target)
             unless resource_paths.empty?
               resource_paths_flattened = resource_paths.flatten.uniq
               input_paths = [script_path, *resource_paths_flattened]
-              output_paths = resource_paths_flattened.map do |input_path|
```

Comment:
This logic is meant for test targets and is duplicated by `target_integrator.rb`, extracted a common method out so we never lose additional fixes again.